docs/guides/session_management_cheerio.ts

```diff
@@ -18,9 +18,9 @@ const crawler = new CheerioCrawler({
         const title = $('title').text();
 
         if (title === 'Blocked') {
-            session.retire();
+            session?.retire();
```

Comment:
Just to be sure, `session` is `undefined` if and only if `useSessionPool: false`, correct? Maybe that'd be worth some type magic in `v4`...

Reply:
https://github.com/apify/crawlee/issues/3280